jquery.flot.js

```diff
@@ -1325,7 +1325,7 @@ Licensed under the MIT license.
             var magn = Math.pow(10, -dec),
                 norm = delta / magn;
 
-            if (norm > 2.25 && norm < 3 && (tickDecimals == null || dec + 1 <= tickDecimals)) {
+            if (tickDecimals != null && norm > 2.25 && norm < 3 && (dec + 1) <= tickDecimals) {
```

Comment:
if tickDecimals is null, dec+1 <= tickDecimals will be false, so there's no need to check != null.